### ISSUE TITLE
Resource aws_cloudwatch_log_metric_filter doesn't support import

### ISSUE THREAD

**Issue by oioki** [terraform-providers/terraform-provider-aws#6042]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I'd like to import current legacy aws_cloudwatch_log_metric_filter into terraform state.

```
$ terraform import aws_cloudwatch_log_metric_filter.filter1 filter1
aws_cloudwatch_log_metric_filter.filter1: Importing from ID "filter1"...

Error: aws_cloudwatch_log_metric_filter.filter1 (import id: filter1): import aws_cloudwatch_log_metric_filter.filter1 (id: filter1): resource aws_cloudwatch_log_metric_filter doesn't support import
```

### New or Affected Resource(s)

* aws_cloudwatch_log_metric_filter

### Potential Terraform Configuration

```
terraform import aws_cloudwatch_log_metric_filter.filter1 cloudwatch_log_group/name
```

### References

https://www.terraform.io/docs/providers/aws/r/cloudwatch_log_metric_filter.html

**Comment by bflad** [comment]
If anyone is interested in helping out with this feature request by contributing to the project, we have recently added a [`Adding Resource Import Support` section to the Contributing Guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#adding-resource-import-support) which has information on how implement the necessary changes. 🎉

**Comment by bflad** [comment]
Support for `aws_cloudwatch_log_metric_filter` resource import has been merged and will release with version 2.55.0 of the Terraform AWS Provider, later this week. Thanks to @holyachon for the implementation. 👍
